CHANGELOG.md

```diff
@@ -27,6 +27,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Wallet
 #### Changed
 - Removed the explicit `id` argument from `Wallet::add_signer()` since that's now part of `Signer` itself
+- Renamed `ToWalletDescriptor::to_wallet_descriptor` to `ToWalletDescriptor::into_wallet_descriptor`
```

Comment:
Perhaps we should mention `ToDescriptorKey` as well?

Reply:
Yes just added and  I also renamed the trait `ToDescriptorKey` to be `IntoDescriptorKey` to match the `into_descriptor_key()` function name.